basemap.py

```diff
@@ -109,20 +110,45 @@ def fetch_quads(mosaic_name, quad_items):
     next = session.get(next_url).json()
     fetch_quads(mosaic_name, next)
 
+def is_overlap(start1, start2, end1, end2):
+  if (max(start1, start2) <= min(end1,end2)):
+    return True
+  else:
+    return False
```

Comment:
replace all 4 lines with a simple

```suggestion
    return max(start1, start2) <= min(end1, end2)
```